sqlite3_community.sql

```diff
@@ -0,0 +1,19 @@
+.open olm_catalog_indexes/index.db.4.6.community-operators
```

Comment:
Why we need these file? Is not the query equals and that just change the image? 
So, why the image cannot be an arg and then, the program extract that and in the go code we do the queries?
IMO that would make easier we use that. Shows hard requires this file. 

I am doing that in the audit. So, it might help you with:
- To pass the image as arg: 
https://github.com/camilamacedo86/audit/blob/main/cmd/bundles/index.go#L33-L34
- To extract the image:
https://github.com/camilamacedo86/audit/blob/main/cmd/bundles/index.go#L104-L123
- To do the queries:
https://github.com/camilamacedo86/audit/blob/main/cmd/bundles/index.go#L126-L153

Reply:
Removed the file in 854a6407 as adding the INDEX_DB_PATH_AND_NAME variable usage removed the need for this example script.